rtengine/iplab2rgb.cc

```diff
@@ -386,6 +386,14 @@ Image16* ImProcFunctions::lab2rgb16b (LabImage* lab, int cx, int cy, int cw, int
 
     Image16* image = new Image16 (cw, ch);
     float p1, p2, p3, p4, p5, p6; //primaries
+    //default primaries
```

Comment:
I think making the ProPhoto branch of the if-else-if-else statements into a pure else so that we always take one of these branches would be preferable to duplicating these values.